packages/run-protocol/src/vaultFactory/prioritizedVaults.js

```diff
@@ -55,7 +53,7 @@ export const makePrioritizedVaults = reschedulePriceCheck => {
   // current high-water mark fires, we reschedule at the new (presumably
   // lower) rate.
   // Without this we'd be calling reschedulePriceCheck() unnecessarily
-  /** @type {Ratio=} */
+  /** @type {Ratio} */
```

Comment:
This change does not preserve semantics.

The `=` for optional is Google Closure syntax, which [TypeScript supports](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns).

Though I think it's fair to standardize on JSDoc syntax which does not support `=`. Instead we'd have to make these `Ratio | undefined`.

Reply:
Having a change of meaning happen automatically is alarming. From your last comment above, we can fix this? Please do and regenerate this PR. Thanks.